decidim-elections/app/models/decidim/votings/voting.rb

```diff
@@ -138,6 +138,10 @@ def has_ballot_styles?
         ballot_styles.exists?
       end
 
+      def check_census_enabled?
+        dataset.present? && !hide_can_i_vote?
```

Comment:
```suggestion
        dataset.present? && show_check_census?
```

Reply:
Specially here it was visible that it's cleaner the `show` approach instead of the `!hide`